cache/cache_test.go

```diff
@@ -490,3 +490,35 @@ func fastForward(r *miniredis.Miniredis, c utils.Clock, secs int64) {
 	r.FastForward(time.Duration(secs) * time.Second)
 	c.Forward(secs)
 }
+
+func TestRedisCacheGetSetCheckInTime(t *testing.T) {
+	r := miniredis.NewMiniRedis()
+	err := r.Start()
+	assert.NoError(t, err)
+	defer r.Close()
+
+	ctx := context.TODO()
+
+	limitsExpSec := 1800
```

Comment:
do you think we can also test for the expiration time?
nit: I am unsure if it is the limits expiration in seconds.

Reply:
I needed some number here to create a cache